### PR TITLE
All bus data for NUS

### LINKED ISSUE #60: Update bus data for entire NUS
Include all stations and buses in NUS. Create a new class BusInfo for Buses.

### PR DESCRIPTION
**Pull request by yuxinwu99**

### CODE REVIEW

File: src/main/java/seedu/duke/Duke.java [hunk 1]
```diff
@@ -41,10 +41,69 @@
...
+        BusStops.UNIVERSITYTOWN,
+        BusStops.KENTVALE
+    };
+    private static final BusStops[] DD2 = {
+        BusStops.KENTVALE,
+        BusStops.UNIVERSITYTOWN,
+        BusStops.COLLEGEGREEN,
+        BusStops.OEITIONGHAM
+    };
+
```
Comment (wamikamalik) [inline src/main/java/seedu/duke/Duke.java:14]: Wondering if we want to move it to a different class like #57

**Review by wamikamalik — APPROVED** [review]
LGTM!

File: src/main/java/seedu/duke/Duke.java (hunk 1 above)
Comment (yuxinwu99) [reply src/main/java/seedu/duke/Duke.java:14]: sure why not

**Comment by Johnson-Yee** [comment]
Looks good to merge!

**Review by EthanWong2212 — APPROVED** [review]
Good to merge

**Pull request merged**